### PR TITLE
Added page id and updated logic for setting parentId of components directly under page

### PR DESCRIPTION
**Pull request by vladbailescu**

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/commons/datalayer/clientlibs/js/datalayer.js [hunk 1]
```diff
@@ -38,13 +38,7 @@
...
 
             if (parentElement.tagName === "BODY") {
-                parentData = dataLayer.find(function(element) {
-                    return element.data !== undefined && element.data.page !== undefined;
-                });
-
-                if (parentData !== undefined) {
-                    component[componentID].parentId = Object.keys(parentData.data.page)[0];
-                }
+                component[componentID].parentId = Object.keys(dataLayer.getState("page"))[0];
```
Comment (jckautzmann) [inline content/src/content/jcr_root/apps/core/wcm/components/commons/datalayer/clientlibs/js/datalayer.js:41]: I would get the page ID directly:
`component[componentID].parentId = parentElement.id`

Comment (vladbailescu) [reply content/src/content/jcr_root/apps/core/wcm/components/commons/datalayer/clientlibs/js/datalayer.js:41]: Then it probably makes no sense to tread `body` elements differently, we could use `component[componentID].parentId = parentElement.id` for both cases, right?

Comment (jckautzmann) [reply content/src/content/jcr_root/apps/core/wcm/components/commons/datalayer/clientlibs/js/datalayer.js:41]: yes, makes sense.

Comment (vladbailescu) [reply content/src/content/jcr_root/apps/core/wcm/components/commons/datalayer/clientlibs/js/datalayer.js]: Done in 8695193

**Pull request merged**